csrc/device_lower/validation.cpp

```diff
@@ -731,6 +731,11 @@ class VectorizeValidator : public OptInDispatch {
       if (!input->isA<TensorView>()) {
         continue;
       }
+      if (tv_def->isA<IndexSelectOp>()) {
```

Comment:
Please add comments

Reply:
I think I need a better validation here. Thanks for catching this one.